apps/neuvector/neuvector/neuvector.yaml

```diff
@@ -68,7 +59,9 @@ spec:
       - '{"config":{"name":"Mta","criteria":[{"key": "address", "value": "mta.reform.hmcts.net", "op": "="}]}}'
       - '{"config":{"name":"Idam","criteria":[{"key": "address", "value": "idam-api.platform.hmcts.net", "op": "="}]}}'
     neuvector:
-      tag: 4.3.2
+      tag: 4.2.0
+      containerd:
+        enabled: true
```

Comment:
no need to add this its the default in the chart:
https://github.com/hmcts/chart-neuvector/blob/master/neuvector-azure-keyvault/values.yaml#L30-L31